src/api/transactions.js

```diff
@@ -246,6 +246,10 @@ export async function getTransactions (ctx) {
     // retrieve transaction request and response bodies
     const transformedTransactions = await addBodiesToTransactions(transactions)
 
+    if (transformedTransactions.orchestrations && transformedTransactions.orchestrations.length > 0) {
```

Comment:
@Zooloo2014 I think we can actually move this logic into the `addBodiesToTransactions` function. So the `addBodiesToTransactions` function will have a transactions parameter for the entire transaction document (request/response/orchestrations/routes etc) and this function will resolve the payloads for all the transaction bodies